nbtools/pylint_notebook.py

```diff
@@ -101,7 +102,10 @@ def pylint_notebook(path=None, options=(), disable=(), enable=(), printer=print,
     pylint_params : dict
         Additional parameter of linting. Each is converted to a separate valid entry in the `pylintrc` file.
     """
-    from pylint import epylint as lint
+    from io import StringIO
+    from pylint.lint import Run
+    from pylint.reporters.text import TextReporter
```

Comment:
`StringIO` can be at the top of the module

Wrap imports from `pylint` into `try/except ImportError` with readable error message